code/obj/item/device/lights.dm

```diff
@@ -481,3 +481,93 @@
 		on = 1
 		set_icon_state(src.icon_on)
 		src.light.enable()
+
+/obj/item/device/light/flare // Absolutely only for use in military situations and station emergencies
```

Comment:
general comment, but I think the flare should be able to heat things under it in the same way candles do.